webdriver/tests/take_element_screenshot/user_prompts.py

```diff
@@ -15,56 +15,100 @@ def take_element_screenshot(session, element_id):
         )
     )
 
+# In some cases, such as "dismiss and notify" & "accept and notify" dismiss
+# the dialog and throw an error. This fixture checks those cases.
+@pytest.fixture
+def check_user_prompt_closed_with_exception(session, create_dialog):
```

Comment:
s/exception/error/g since we probably want to use the WebDriver terminology here.

Reply:
Fixed